runners/apex/src/main/java/org/apache/beam/runners/apex/ApexRunner.java

```diff
@@ -258,9 +260,15 @@ protected String getKindString() {
             PCollection<InputT>, PCollectionView<OutputT>,
             Combine.GloballyAsSingletonView<InputT, OutputT>> {
       @Override
-      public PTransform<PCollection<InputT>, PCollectionView<OutputT>> getReplacementTransform(
-          GloballyAsSingletonView<InputT, OutputT> transform) {
-        return new StreamingCombineGloballyAsSingletonView<>(transform);
+      public PTransformReplacement<PCollection<InputT>, PCollectionView<OutputT>>
+          getReplacementTransform(
+              AppliedPTransform<
+                      PCollection<InputT>, PCollectionView<OutputT>,
+                      GloballyAsSingletonView<InputT, OutputT>>
+                  transform) {
+        return PTransformReplacement.of(
```

Comment:
There's definitely utilities we can add. Due to the reason this PR exists, they need to handle an iterable that contains more than a single input, where only one is the "main input", so I've migrated to that everywhere.